db/mongodb_test.go

```diff
@@ -106,6 +107,25 @@ func TestMongoDB(t *testing.T) {
 	if err := db.CreateExtraIndexes([]string{"teste.index1"}); err == nil {
 		t.Error("expected errors running extra indexes, got nil")
 	}
+	q := Query{Uf: "SP", Limit: 1}
+	_, err = db.Search(q)
+	if err != nil {
+		t.Errorf("expected no error getting query, got %s", err)
+	}
+	q = Query{Uf: "SP", Limit: 1}
+	sr, _ := db.Search(q)
```

Comment:
Pq dois `q` com o mesmíssimo valor? Pq duas chamadas iguais a `Search`?

```suggestion
	q := Query{Uf: "SP", Limit: 1}
	sr, err = db.Search(q)
	if err != nil {
		t.Errorf("expected no error getting query, got %s", err)
	}
```

Reply:
Uma era para ser outro UF para simular uma retorno nulo  ( para vir  data: nil, cursor: nil )